homeassistant/components/deconz/fan.py

```diff
@@ -100,7 +112,7 @@ def __init__(self, device, gateway) -> None:
     @property
     def is_on(self) -> bool:
         """Return true if fan is on."""
-        return self._device.speed != FAN_SPEED_OFF
+        return bool(self._device.speed != FAN_SPEED_OFF)
```

Comment:
This change is not needed, the result of this expression is already a boolean.

```suggestion
        return self._device.speed != FAN_SPEED_OFF
```

Reply:
Fixed